app/scripts/background.js

```diff
@@ -327,7 +327,7 @@ function setupController (initState, initLangCode) {
     const isMetaMaskInternalProcess = metamaskInternalProcessHash[processName]
 
     if (metamaskBlockedPorts.includes(remotePort.name)) {
-      return false
+      return
```

Comment:
This is potentially functionally different. Why is this the correct way to fix this lint error?

Reply:
I suspect it's because the return value is not used by an caller.

Reply:
Because this function is only used as a listener for `extension.runtime.onConnect`:

```js
extension.runtime.onConnect.addListener(connectRemote)
```

And the return value from the event listener isn't conventionally important. It's also not documented as being used.<sup>[\[1\]][1]</sup>

  [1]:https://developer.chrome.com/extensions/runtime#event-onConnect